base.php

```diff
@@ -2234,7 +2234,8 @@ function unload($cwd) {
 	*	@return mixed
 	*	@param $key string
 	**/
-	function offsetexists($key) {
+        #[\ReturnTypeWillChange]
+        function offsetexists($key) {
```

Comment:
This changes the indentation from tabs to spaces. Same issue on most other changed lines.

Reply:
Sorry! I think I fixed it now :)

Reply:
There are still several instances. You can use [Refined GitHub](https://github.com/refined-github/refined-github) extension to make the whitespace visible.

Reply:
I fixed the indentation as well as added descriptions in the commit messages and few more fixes in #333.